.travis.yml

```diff
@@ -81,15 +87,15 @@ before_script:
    # Now build pFUnit
    - cd ${TRAVIS_BUILD_DIR}
    - mkdir -p build && cd build
-   - cmake .. -DCMAKE_Fortran_COMPILER=${FC} -DCMAKE_INSTALL_PREFIX=${HOME}/Software/pFUnit
+   - cmake .. -DCMAKE_Fortran_COMPILER=${FC} -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=${HOME}/Software/pFUnit
```

Comment:
Note pFUnit uses debug flags regardless of CMAKE_BUILD_TYPE.    I could not see any good reason to ever build without them for this utility.   Even my most expensive project with something like 2500 tests runs them all in a couple of seconds.

Reply:
I will remove it.

Reply:
Done